.pa11yci.conf.js

```diff
@@ -11,7 +11,7 @@ var config = {
   urls: [
     {
       url: '${HOST}/login',
-      wait: 500,
+      wait: 1000,
```

Comment:
is this necessary for more stability?

Reply:
Great questions Ash, we think that the contrast errors are caused because of the background not having enough time to load.

Reply:
But looking at the results it didn't have any effect so the errors on `/login/` still need some more investigation. I'll look into that.